src/test/java/ay2122s1_cs2103t_w16_2/btbb/exception/CommandExceptionTest.java

```diff
@@ -0,0 +1,20 @@
+package ay2122s1_cs2103t_w16_2.btbb.exception;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class CommandExceptionTest {
+    private static final String COMMAND_EXCEPTION_MESSAGE = "This is a CommandException.";
+
+    @Test
+    public void constructor_throwCommandException_commandExceptionThrown() {
+        assertThrows(CommandException.class, () -> {
+            throw new CommandException(COMMAND_EXCEPTION_MESSAGE);
+        });
+
+        assertThrows(CommandException.class, () -> {
+            throw new CommandException(COMMAND_EXCEPTION_MESSAGE, new Throwable());
+        });
```

Comment:
Also I'm not sure if there is a point in asserting that CommandException.class == CommandException.class? But I suppose it can be kept just for coverage? What do you think?

Reply:
All the test for the exceptions are not really needed. I only added to increase coverage.